src/components/bulk-email-tool/bulk-email-form/TaskAlertModal.jsx

```diff
@@ -35,7 +35,9 @@ function TaskAlertModal(props) {
               id="bulk.email.form.recipients.Contine"
               defaultMessage="Continue"
               description="Continue button for the task alert"
-            />
+            >
+              {msg => <>{msg}</>}
```

Comment:
nit: This looks a bit cryptic. Is there any way we'd be able to add a comment here/nearby that explains why we have to do this? I'm not sure I would remember why we did this in a few months time.

Reply:
absolutely! comment added!